base/ltfssaxes.dtx

```diff
@@ -905,12 +905,8 @@
 %    \end{macrocode}
 %
 %    \begin{macrocode}
-\DeclareFontSeriesChangeRule {m}{l}{l}  {m}                 %<-----
 \DeclareFontSeriesChangeRule {m}{sb}{sb}  {b}               %<-----
 \DeclareFontSeriesChangeRule {m}{b}{b}  {bx}                %<-----
-\DeclareFontSeriesChangeRule {m}{ec}{ec}  {m}               %<-----
-\DeclareFontSeriesChangeRule {m}{c}{c}  {m}                 %<-----
-\DeclareFontSeriesChangeRule {m}{sc}{sc}  {m}               %<-----
 \DeclareFontSeriesChangeRule {m}{bx}{bx}  {b}               %<-----
```

Comment:
There are a number of entries of the form `\DeclareFontSeriesChangeRule{...}{...}{bx}{b}`. I suspect that they exist because for the Computer Modern fonts 'bold' is `bx` and not `b`, so that you have a reasonable substitution when `bx` is applied to another font. Wouldn't it be better if this was handled in a general way, e.g., in `\merge@font@series@`, instead of handling this case (incompletely) via random `\DeclareFontSeriesChangeRule` entries?